content/updates/2020-07/1-easily-fix-errors-on-your-connections.md

```diff
@@ -0,0 +1,15 @@
+# Easily fix errors on your connections 
+::: meta-data type="Improvement" features="Connections, Productivity" available="early-July" link-to-doc="../articles/connection-dashboard.html#in-the-connection-monitoring-page"
+:::
+
+We improved the connection error messages by adding several information that should help you to fix them.
+
+In the `Monitoring` page, you will now find links to other PIM page or help center pages, and some data about the related product.  
+
+![Enriched error message](../img/fix-errors-on-your-connections.png)
```

Comment:
The image is more than 793px wide. Also, text on the screenshot is cropped, is it on purpose?